src/utils/graphql/fragments/fullTemplate.fragment.ts

```diff
@@ -2,7 +2,7 @@ import { gql } from '@apollo/client'
 
 export default gql`
   fragment FullTemplate on Template {
-    ...Template
+    ...templateFragment
```

Comment:
Is this consistent with how we've been naming other fragments?

Reply:
New ones yes, old ones are like it was before `Template`, but I think it's not very clear where in use. So preferred the new way, do you agree?

Reply:
Yeah sure. Is the convention to use camelCase for these, or would it be "TemplateFragment" ?